eng/common/TestResources/New-TestResources.ps1

```diff
@@ -154,14 +154,21 @@ if ($ProvisionerApplicationId) {
     $subscriptionArgs = if ($SubscriptionId) {
         @{SubscriptionId = $SubscriptionId}
     }
+    else {
```

Comment:
These changes are a reflection of those made in [#828](https://github.com/Azure/azure-sdk-for-net/pull/828) in the tools repository.  I included them here because the sync does not appear to have happened yet.  

It is expected that this file will be overwritten when the tools repository is sync'd.

Reply:
I just merged the sync PR https://github.com/Azure/azure-sdk-for-net/pull/13786

Reply:
Thanks, Wes.   I'll pull this out of here then!